tests/DCPS/DomainRange/DomainRangeTest.mpc

```diff
@@ -0,0 +1,7 @@
+project: dcpsexe, dcps_test, dcps_transports_for_test {
+  exename = DomainRangeTest
+  idlflags += -SS
+  TypeSupport_Files {
+    TestMsg.idl
```

Comment:
Could this use `tests/DCPS/ConsolidatedMessengerIdl` to avoid needing code generation unique to this test?

Reply:
I modified the test to use ConsolidatedMessengerIdl